### PR TITLE
e2e common cdap replication steps

### PR DESCRIPTION
**Pull request by itsmekumari**
e2e common cdap replication steps

### CODE REVIEW

File: src/main/java/io/cdap/e2e/pages/actions/CdfPluginPropertiesActions.java [hunk 1]
```diff
@@ -987,4 +991,116 @@ public static void replaceValueInInputPropertyFromEnv(String pluginProperty, Str
...
+    SeleniumDriver.getDriver().switchTo().window(parentWindow);
+    ElementHelper.clickOnElement(CdfPluginPropertiesLocators.stop);
+  }
+
+  /**
+   * Verify Error message displayed on the header/at the top of the Plugin Properties page using the Error message
+   * location in the .properties file {@link ConstantsUtil#DEFAULT_ERROR_PROPERTIES_FILE}
+   *
+   * @param errorMessage Expected error message location
+   */
```
Comment (MrRahulSharma) [inline src/main/java/io/cdap/e2e/pages/actions/CdfPluginPropertiesActions.java]: The function comments matches with the existing method `verifyErrorMessageOnHeader` please update the function comment specific to this method.

**Review by MrRahulSharma — APPROVED** [review]
LGTM, with minor comments.

**Review by Aryan-Verma — APPROVED** [review]

File: src/main/java/io/cdap/e2e/pages/actions/CdfPluginPropertiesActions.java (hunk 1 above)
Comment (itsmekumari) [reply src/main/java/io/cdap/e2e/pages/actions/CdfPluginPropertiesActions.java]: Change Done.

File: src/main/java/io/cdap/e2e/pages/actions/CdfPluginPropertiesActions.java [hunk 2]
```diff
@@ -987,4 +991,116 @@ public static void replaceValueInInputPropertyFromEnv(String pluginProperty, Str
...
+  public static void runTheReplicationPipeline() {
+    startReplicationPipeline();
+    WaitHelper.waitForElementToBeDisplayed(CdfPluginPropertiesLocators.running);
+  }
+
+  /**
+   * Open Advanced logs while pipeline is in running state
+   */
+  public static void openAdvancedLogs() {
+    CdfPluginPropertiesLocators.logs.click();
```
Comment (itsankit-google) [inline src/main/java/io/cdap/e2e/pages/actions/CdfPluginPropertiesActions.java:1041]: why not using element helper here?

**Pull request merged**